content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -134,15 +135,16 @@ log audit backend using the following `kube-apiserver` flags:
 - `--audit-log-maxbackup` defines the maximum number of audit log files to retain
 - `--audit-log-maxsize` defines the maximum size in megabytes of the audit log file before it gets rotated
 
-In case kube-apiserver is configured as a Pod,remember to mount the hostPath to the location of the policy file and log file. For example, 
-`
---audit-policy-file=/etc/kubernetes/audit-policy.yaml
---audit-log-path=/var/log/audit.log
-`
+If your cluster's control plane runs the kube-apiserver as a Pod, remember to mount the `hostPath`
```

Comment:
nit: Is there a single location for the policy and log files?
nit: Line 141 ,What are the `...` before the first audit flag? Move to a different line?

Reply:
> Is there a single location for the policy and log files?

I haven't checked.
I'll remove the “...”